src/modules/fancyzones/tests/UnitTests/ZoneWindow.Spec.cpp

```diff
@@ -537,7 +537,7 @@ namespace FancyZonesUnitTests
             Assert::AreEqual(expected, actual);
 
             const auto zoneSet = zoneWindow->ActiveZoneSet();
-            zoneSet->MoveWindowIntoZoneByIndex(window, Mocks::Window(), 0);
+            zoneSet->MoveWindowIntoZoneByIndex(window, Mocks::Window(), 1);
```

Comment:
I think you don't need to change the three numbers in this file

Reply:
You're right. Done

Reply:
I don't see the commit

Reply:
push was rejected for some reason. It's there now :)